models/labels/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql

```diff
@@ -0,0 +1,140 @@
+{{
+    config(
+        alias='likely_bot_addresses',
+        post_hook='{{ expose_spells(\'["optimism"]\', 
+        "sector", 
+        "labels", 
+        \'["msilb7"]\') }}'
+    )
+}}
+
+
+-- This could/should become a spell with some kind of modular logic approach so that we can plug in new detection logic over time (i.e. many of X method, or Y project's contracts)
+-- the core of this logic is on transaction frequency and sender concentration The "sender concentration" piece will get tested by mutlisigs / smart contract wallets.
+
+WITH  bot_addrs AS (
```

Comment:
Would you mind reformatting as CTEs instead of nested subqueries? I think it will be easier to debug and edit the logic as needed.

Reply:
done